### DISCUSSION TITLE
FR: Cmd + Shift + Plus / Minus to increase/decrease font size consistently

### DISCUSSION THREAD

**Discussion by thiagowfx** [ghostty-org/ghostty#3888] [Feature Requests, Ideas] [answered]
Out-of-the-box, the shortcut to increase/decrease font sizes in Ghostty on macOS is `Cmd + +` and `Cmd + -`, respectively (no Shift).

With Shift:

**Observed**: `Cmd + Shift + +` inserts a literal '+', and `Cmd + Shift + -` decreases the font size.

**Expected**: `Cmd + Shift + +` should increase the font size as well, consistently.

Why is the behavior for `Cmd + Shift + +` different? This seems unintentional.

Note that this is the out-of-the-box behavior in both Terminal.app and iTerm2.

**Comment by mitchellh** [comment, marked as answer]
Approved #3899
